semperfiwebdesign/all-in-one-seo-pack
# Add logo to Welcome screen

Issue #477

## Proposed changes

Adds our gear logo to the welcome screen.

![image](https://user-images.githubusercontent.com/18156697/60727638-ab2aac00-9f3e-11e9-99a6-9d9c772b6627.png)

I've also moved the SVG for the gear logo over to a dedicated function that allows us to specify the dimensions and fill colour so that it can be used dynamically, added a small RTL CSS fix and added the "aioseop" prefix to the relevant CSS files.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

## Checklist


## Testing instructions
- Go to the Welcome screen (/wp-admin/index.php?page=aioseop-about) and make sure it looks good both in LTR and RTL languages.
- Make sure the menu icon in the toolbar still looks good since the function for it has been refactored.
